python/delta/tests/test_deltatable.py

```diff
@@ -977,6 +1022,39 @@ def test_delta_table_builder_with_bad_args(self) -> None:
         with self.assertRaises(TypeError):
             builder.addColumn("a", "int", generatedAlwaysAs=1)  # type: ignore[arg-type]
 
```

Comment:
For parity with the Scala APIs, there are some negative tests missing here: the column data type must be Long/bigint.